test/definition/ApiDefinitionFactorySpec.scala

```diff
@@ -79,12 +79,12 @@ class ApiDefinitionFactorySpec extends UnitSpec {
                   version = Version3,
                   status = BETA,
                   endpointsEnabled = true
-                ),
-                APIVersion(
-                  version = Version4,
-                  status = BETA,
-                  endpointsEnabled = true
                 )
+//                APIVersion(
+//                  version = Version4,
+//                  status = BETA,
+//                  endpointsEnabled = true
+//                )
```

Comment:
Perhaps leave an explanation. Something along the lines of:
```suggestion
// To be reinstated, see MTDSA-20412
//                APIVersion(
//                  version = Version4,
//                  status = BETA,
//                  endpointsEnabled = true
//                )
```
See [MTDSA-20412](https://jira.tools.tax.service.gov.uk/browse/MTDSA-20412)